apps/dashboard/src/app/(dashboard)/dashboard/connect/ecosystem/[ecosystemId]/layout.tsx

```diff
@@ -5,6 +5,7 @@ export default function Layout({
   children,
   params,
 }: { children: React.ReactNode; params: { ecosystemId: string } }) {
+  // TODO: check if the user has an access token, if they don't redirect them
```

Comment:
useLoggedInUser should help here, right?

Reply:
This is server-side and I ideally don't want to bring it to the client so I can add metadata